src/public/Interfaces.ts

```diff
@@ -75,16 +81,25 @@ export interface Context {
     res?: { [key: string]: any };
 }
 
-export interface HttpRequestHeaders {
-  [name: string]: string | undefined;
+/**
+ * HTTP request headers.
+ */
+export interface HttpHeaders {
+  [name: string]: string | unknown;
```

Comment:
I think this is actually `[name: string]: string`

Headers, query, and params will always be a string (even if that string is `""`) per this change: https://github.com/Azure/azure-functions-nodejs-worker/commit/544f7e028409d8be54de85d81ba69d4dbeaf5138

Reply:
Any reason in particular to change this from `HttpRequestHeaders` => `HttpHeaders`?

Reply:
EDIT: Should remain `[name: string]: string | undefined`, I'm wondering if it should remain HttpRequestHeaders too!